<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/ladybug.png</normaloff>icon/ladybug.png</iconset>
  </property>
  <widget class="QLabel" name="login_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>icon/login.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>700</y>
     <width>600</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/newPrefix/invisible.png); border: 0px
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>500</y>
     <width>930</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Baskerville BT</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
background:transparent;
border:non;
}</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::PasswordEchoOnEdit</enum>
   </property>
   <property name="cursorPosition">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserID">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>270</y>
     <width>930</width>
     <height>90</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
background:transparent;
border:non;
}</string>
   </property>
   <property name="text">
    <string>User ID</string>
   </property>
   <property name="cursorPosition">
    <number>7</number>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="cursorMoveStyle">
    <enum>Qt::LogicalMoveStyle</enum>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon/back.qrc"/>
 </resources>
 <connections/>
</ui>
